etcdserver/api/v2http/client.go

```diff
@@ -303,7 +303,7 @@ func (h *membersHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		case err == membership.ErrIDNotFound:
 			writeError(w, r, httptypes.NewHTTPError(http.StatusNotFound, fmt.Sprintf("No such member: %s", id)))
 		case err != nil:
-			plog.Errorf("error updating member %s (%v)", m.ID, err)
+			plog.Errorf("error updating member %s in %v (%v)", m.ID, h.timeout, err)
```

Comment:
this is bad. the error might not be a timeout... if it is not a timeout, we should not print out the timeout in seconds.